style.css

```diff
@@ -0,0 +1,97 @@
+/* stylelint-disable length-zero-no-unit */
+body {
+  background-color: #f7f7f9;
+  margin: 0px;
```

Comment:
Great job fixing the suggestion of the previous requirement. However, it is one of the general requirements to not disable linter as they are there for a reason. 
![image](https://user-images.githubusercontent.com/59416136/153263085-02a029c1-2eb6-446c-a443-a209327aea5f.png)

------

- [ ] Kindly remove stylelint disable and fix any errors that come up. You can read this [article](https://stylelint.io/user-guide/rules/list/length-zero-no-unit/) for further explanation.
------

Reply:
Well, I haven't changed the configs, I just told it to ignore that part. I re-submitted again tho.